content/desktop/use-desktop/builds.md

```diff
@@ -41,33 +26,126 @@ active builders. The list doesn't include builds for inactive builders.
 After the restart, the **Builds** view and the **Builders** settings menu
 appear.
 
+## Show build list
+
+Select the **Builds** view in the Docker Dashboard to open the build list.
+
+The build list shows running and completed builds for your active builders. The
+top-right corner shows the name of your currently selected builder, and the
+**Builder settings** button lets you [manage builders](#manage-builders) in the
+Docker Desktop settings.
+
+Running builds are displayed in the top section of this view, with information
+about the build target and progress.
+
+The lower section of the view shows your recent builds. You can select any of
+the builds from the active builds or completed builds section to view detailed
+information, such as logs, a performance breakdown, cache utilization, and
+more.
+
+The build list doesn't include builds for inactive builders. That is, builders
+that you've removed from your system, or builders that have been stopped.
+
 ## Inspect a build
 
 To inspect a build, select the build that you want to view in the list.
+The inspection view contains a number of tabs:
+
+### Info tab
+
+The **Info** tab displays details about the build.
+
+If you're inspecting a multi-platform build, the drop-down menu in the
+top-right of this tab lets you filter the information down to a specific
+platform:
+
+![Platform filter](../images/build-ui-platform-menu.webp?w=400)
+
+The **Source details** section shows the filepath to the
+[frontend](../../build/dockerfile/frontend.md) that was used and, if available,
+information about the source code repository.
+
+The **Build timing** section contains charts showing a breakdown of the build
+execution from various angles.
 
-The **Info** tab displays details about the build job. The details include
-information such as target stage for multi-stage builds, target platforms, and
-version control information, if available.
+- **Real time** refers to the wall-clock time that it took to complete the build.
+- **Accumulated time** shows the total CPU time for all steps.
+- **Cache usage** shows the extent to which build operations were cached.
+- **Parallel execution** shows how much of the build execution time was spent running steps in parallel.
+
+![Build timing charts](../images/build-ui-timing-chart.webp)
+
+The chart colors and legend keys describe the different build operations. Build
+operations are defined as follows:
+
+| Build operation      | Description                                                                     |
+| :------------------- | :------------------------------------------------------------------------------ |
+| Local file transfers | ???                                                                             |
+| File operations      | ???                                                                             |
```

Comment:
File operations: Any llb.Copy, llb.Mkfile, llb.Mkdir so for Dockerfile frontend it means `COPY`, `WORKDIR`, `ADD` (for files)

Reply:
How about `RUN mkdir -p hey` etc?

Reply:
No this is for `Executions` (`RUN`)